### ISSUE TITLE
gap reason="erasure"

### ISSUE THREAD

**Issue by thea-m** [SChAth/ScAthiop#496]
`gap reason="erasure"` is no longer accepted by the schema - should it be added or should we use `del rend="erasure"` instead?

**Comment by PietroLiuzzo** [comment]
It depends what is that you want to encode. if there is a gap or a deletion.... There are only 4 instances of `gap[@reason='erasure']`, and it is not documented in the [guidelines](
https://github.com/SChAth/ScAthiop/wiki/guidelines#in-text) what would be the distinction?

**Comment by thea-m** [comment]
you're right, our occurences of `gap reason="erasure"` should be in fact `del rend="erasure"`. I corrected it in my records, could you check and correct BAVet33, BAVet49 and BAVet54, @MassimoVilla ?

**Comment by MassimoVilla** [comment]
done. Thank you Thea
